OmniSharp.sln

```diff
@@ -68,6 +69,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OmniSharp.Stdio.Driver", "s
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OmniSharp.Script.Tests", "tests\OmniSharp.Script.Tests\OmniSharp.Script.Tests.csproj", "{9E4BA68C-7F4B-429A-A0C7-8CE7D41D610F}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OmniSharp.MiscellaneousFile", "src\OmniSharp.MiscellaneousFile\OmniSharp.MiscellaneousFile.csproj", "{23A91D15-B178-45A9-8F7A-B0CE1F7A8DAA}"
```

Comment:
why is the project + folder called `OmniSharp.MiscellaneousFile` (singular) but everywhere we refer to  `MiscellaneousFiles`? I believe for consistency, the project and folder should be renamed to `OmniSharp.MiscellaneousFiles` too